### ISSUE TITLE
schematic overlays do not render when tping

### ISSUE THREAD

**Issue by end-user** [maruohon/litematica#564]
I recently updated to MC v1.19.2, malilib v0.13.0, litematica v0.12.4.  I can load and see schematic overlays just fine.  However, when I tp away and tp back (even in single player), the overlay is missing.  I don't see a render toggle option, so I left with going into the placements menu, disabling and re-enabling it.  This is necessary for _each_ placement.

**Comment by aditeya24** [comment]
What happened is that the schematics got unloaded. You can reload them by turning off schematic rendering (M+G) or all rendering (M+R) and turn it on again.

**Comment by maruohon** [comment]
I think this is likely the recently introduced(?) compatibility issue when Starlight is also installed. Re-loading schematic chunks doesn't either re-build the chunk or doesn't update the renderer, I haven't looked into it yet.

**Comment by eat-your-glory** [comment]
The schematic probably unloaded

Il giorno mar 23 ago 2022 alle 02:04 Oblio ***@***.***> ha
scritto:

**Comment by petersv5** [comment]
M+R or M+G to toggle the schematic / global rendering does not bring the schematic rendering back for me. Disabling/reenabling the individual placement or using the "R" button on the placement config screen does work. 

The issue with starlight has been there for all 1.19 releases afaik. It was not present for any of the 1.18 releases.

**Comment by end-user** [comment]
Agreed - I _am_ using Starlight, but my version 1.1.1 did not change when I updated to 1.19.2 (and the accompanying litematica update).  Not saying it's _not_ a mod conflict, just saying the Starlight code hasn't changed.

Also wanted to agree that it looks like schematics not getting loaded with the chunk.  If I tp within the loaded render distance they remain.  I wanted to ammend that, if I have two schematics that are not showing, I only need to disable one to get the other to render.  (then re-enable it of course; just pointing out that the "fix" happens at disable)

**Comment by Rektroth** [comment]
This issue is happening for me as well when Starlight is installed.

Like petersv5 reports, M+R and M+G do **not** reload the schematic - only disabling and then re-enabling the placement does.

**Comment by maruohon** [comment]
The "pre" version here should fix this issue, see if it works in your case: <https://masa.dy.fi/mcmods/client_mods/?mcver=1.19.2>

**Comment by petersv5** [comment]
The pre-version linked above solves the issue for me at least.
Thanks!

**Comment by Rektroth** [comment]
Issue is fixed in the pre version. 👍
